CompanyEmployees.Infrastructure.Persistence/Configurations/AttendanceConfiguration.cs

```diff
@@ -0,0 +1,28 @@
+using CompanyEmployees.Core.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace CompanyEmployees.Infrastructure.Persistence.Configurations;
+
+public class AttendanceConfiguration : IEntityTypeConfiguration<Attendance>
+{
+    public void Configure(EntityTypeBuilder<Attendance> builder)
+    {
+        builder.HasKey(a => a.Id);
+
+        builder.HasOne(a => a.Employee)
```

Comment:
Consider explicitly specifying the navigation collection on the `Employee` entity using `.WithMany(e => e.Attendances)` (if such a property exists) to improve code clarity and make the relationship more explicit.

```suggestion
.WithMany(e => e.Attendances)
```